template/sample/template.gen.go

```diff
@@ -152,6 +174,15 @@ var (
 					}
 				}
 
+				for _, v := range cpb.BoolMap {
+					if t, e := tEvalFn(v); e != nil || t != istio_mixer_v1_config_descriptor.BOOL {
```

Comment:
I think we need to move istio_mixer_v1_config_descriptor to a new namespace... Unrelated to this PR but should be done for 0.2